it_IT/ODF-community.md

```diff
@@ -362,41 +360,41 @@ Risorse
 ===========================================================================
 _page 12 of 15_ 
 
-  - [Red Hat Multiplier](https://github.com/red-hat-people-team/red-hat-multiplier) – breve promemoria su collaborazione, transparenza, fiducia, meritocrazia, connessione
-  - [The Open Source Way handbook](https://www.theopensourceway.org/wiki/Introduction) –  guida per creare e supportare comunità di contributori
+  - [Red Hat Multiplier](https://github.com/red-hat-people-team/red-hat-multiplier) – promemoria veloce su collaborazione, transparenza, fiducia, meritocrazia, connessione
+  - [The Open Source Way handbook](https://www.theopensourceway.org/wiki/Introduction) –  guida per la creazione e il mantenimento di comunità di contributori
   - [The Open Organization](http://opensource.com/open-organization) (libro + comunità online)
   - [Prioritizing by impact](http://developerblog.redhat.com/2014/08/27/5-ux-tips-for-developers/), vedere la tabella nell'articolo "5 UX Tips for Developers" di Máirín Duffy
-  - [Opensource.com](http://www.opensource.com/) – Una pubblicazione supportata da Red Hat e focalizzata su come i principi open source possono essere applicati al business, all'istruzione, al governo ecc.
+  - [Opensource.com](http://www.opensource.com/) – Una pubblicazione supportata da Red Hat e focalizzata su come i principi open source possono essere applicati al business, all'educazione, al governo ecc.
   - [The Advice Process](http://danieltenner.com/2014/11/06/the-advice-process-definition-and-usage-tips/) (Daniel Tenner)
 
 
 Appendici 
 ===========================================================================
 
-Storia: Come nasce l'Open Decision Framework
+Storia: Da dove proviene l'Open Decision Framework
 ----------------------------------------------------
 _page 14 of 15_ 
 
   - Basato su principi praticati dalle comunità open source
 
-    - Ricerca della Duke University's Fuqua School of Business e di Diana Martin (2009 – 2010); ulteriori risorse dalla comunità
+    - Ricerca della Duke University's Fuqua School of Business e di Diana Martin (2009 – 2010); risorse addizionali della comunità
 
   - Sviluppato dal People team, con contributi da un focus group interfunzionale
 
-    - È il risultato dell'impegno del Project Management Office (People team) per creare una metodologia aperta di project management (2012 – 2013)
-    - Le conversazioni su memo-list in merito al rilascio di Google Calendar sono servite per condividere bozze con tutti i collaboratori e invitare alla partecipazione (2014)
-    - È stato testato dal team IT e Engineering nel gruppo di lavoro sul passaggio a Google Calendar (2014 – 2015)
```

Comment:
È stato testato dai team IT e Engineering, che hanno collaborato allo sviluppo di una soluzione di bridging per Google Calendar (2014 – 2015)